reference/docs-conceptual/setup/Installing-PowerShell-Core-on-Linux.md

```diff
@@ -50,7 +50,8 @@ sudo apt-get install -y powershell
 pwsh
 ```
 
-As superuser, Register the Microsoft repository. From then on, you just need to use `sudo apt-get upgrade powershell` to update the installation.
+As superuser, register the Microsoft repository. From then on, you just need to use
```

Comment:
The second sentence should be on a separate line

Reply:
fixed